chain/docker-config/generate-genesis.sh

```diff
@@ -35,12 +35,11 @@ expect $DIR/add-account.exp \
     "smile stem oven genius cave resource better lunar nasty moon company ridge brass rather supply used horn three panic put venue analyst leader comic" \
 
 # add accounts to genesis
-bandd add-genesis-account $(bandcli keys show validator1 -a) 10000000000000uband
-bandd add-genesis-account $(bandcli keys show validator2 -a) 10000000000000uband
-bandd add-genesis-account $(bandcli keys show validator3 -a) 10000000000000uband
-bandd add-genesis-account $(bandcli keys show validator4 -a) 10000000000000uband
-
-bandd add-genesis-account $(bandcli keys show requester -a) 10000000000000uband
+bandd add-genesis-account validator1 10000000000000uband --keyring-backend test
```

Comment:
what is the `test` here -- is it a flag/option or user-defined string that we hard-coded here?

Reply:
It's the configuration of where to store & encrypted key (mnemonic).